README.md

```diff
@@ -67,22 +100,20 @@ Options:
    --tls-san [--tls-san option --tls-san option]							Support extra SANs for TLS certs
 ```
 
-### 2. Create your machine
+## Development
+
+### Build from source
+
+If you wish to work on this driver, you will first need Go installed. Make sure Go is properly installed, including setting up a [GOPATH](https://golang.org/doc/code.html#GOPATH).
+
+Run these commands in root of repository to build the plugin binary:
 
 ```shell
-$ docker-machine create --driver hyperone --hyperone-token TOKEN --hyperone-project PROJECT vm
-Running pre-create checks...
-Creating machine...
-(vm) Creating HyperOne VM...
-Waiting for machine to be running, this may take a few minutes...
-Detecting operating system of created instance...
-Waiting for SSH to be available...
-Detecting the provisioner...
-Provisioning with debian...
-Copying certs to the local machine directory...
-Copying certs to the remote machine...
-Setting Docker configuration on the remote daemon...
-Checking connection to Docker...
-Docker is up and running!
-To see how to connect your Docker Client to the Docker Engine running on this virtual machine, run: docker-machine env vm
+$ go build
```

Comment:
I would include previous commands as well, as this seems to be an instruction for a newcomer to Go world:

```shell
$ go get ....
$ cd $GOPATH/.....
$ go build
```

Reply:
@gregorybrzeski , ```go build``` automatically fetch dependencies. There is no need to use "go get" if the command is executed from the root of repository

Reply:
How do you get the the repository in which you execute the `go build` then ? IMHO that step is missing here.